doculab/docs/api-duplicate-prevention.textile

```diff
@@ -0,0 +1,59 @@
+When making API requests, it is possible to receive an error even though your request actually completed successfully.
+
+For example, if you receive a @502 Bad Gateway@ or @504 Gateway Timeout@ error, you can't be sure whether your request was received by Chargify, or not.
+
+If you re-try the request after receiving one of these errors, you _might_ end up with a duplicate transaction.
+
+h2. Uniqueness Token
+
+In order to prevent these duplicates, Chargify allows you to supply a @uniqueness_token@ parameter in any POST or PUT request.
+
+The value you supply for the @uniqueness_token@ parameter is up to you.  We recommend using a long, random value, like a UUID.
```

Comment:
I'd like to reword this to say "The value should be a long, random value, like a UUID.  The exact **format** of the value is up to you.